catroid/src/main/java/org/catrobat/catroid/content/ListWithoutDuplicates.java

```diff
@@ -121,6 +121,8 @@ public boolean retainAll(Collection<?> c) {
 	public T set(int index, T element) {
 		try {
 			if (element.equals(this.get(index))) {
+				comparingSet.clear();
```

Comment:
unrelated to the two lines that were added here: The catch block of the enclosing try block is just rethrowing and should be omitted. This is also not the only occurrence of a catch block that only rethrows an exception, please also remove the others.